sentinelhub/geopedia.py

```diff
@@ -248,3 +248,12 @@ def get_field_iterator(self, field):
         """
         for feature in self:
             yield feature['properties'].get(field, [])
+
+    def add_query(self, query=None):
+        """ Add query, e.g. f12345 = 2015
+
+        :param query: query string
+        :type query: str
+        """
+        if query:
+            self.query['filterExpression'] += ' && ' + query
```

Comment:
It is possible that `self.query` doesn't have `'filterExpression'` key. Therefore I suggest doing something like:
```Python
self.query['filterExpression'] = '{} && {}'.format(self.query['filterExpression'], query) if self.query.get('filterExpression') else query
```
And since you are repeatedly using string `'filterExpression'` I suggest declaring it in a constant inside the class.

Reply:
It seems not possible to me, as `self.query['filterExpression']` is set in `__init__`. Otherwise, I agree.